internal/executor/vaultunboxer/boxedvalue.go

```diff
@@ -44,10 +45,13 @@ func NewBoxedValue(rawBoxedValue string) (*BoxedValue, error) {
 	}
 
 	parts := strings.Split(rawBoxedValue, " ")
-	if len(parts) != 2 {
-		return nil, fmt.Errorf("%w: there should be 2 parameters (path and a selector), found %d",
+	if len(parts) < 2 {
+		return nil, fmt.Errorf("%w: there should be at least 2 parameters (path and a selector), found %d",
 			ErrInvalidBoxedValue, len(parts))
 	}
+	if strings.Contains(parts[1], "=") {
+		return nil, fmt.Errorf("%w: missing selector element, found '%q'", ErrInvalidBoxedValue, parts[1])
```

Comment:
%q already quotes the value:

```suggestion
		return nil, fmt.Errorf("%w: missing selector element, found %q", ErrInvalidBoxedValue, parts[1])
```